net/browserless_agent/src/browserless_agent.ts

```diff
@@ -0,0 +1,202 @@
+import { AgentFunction, AgentFunctionInfo, DefaultConfigData } from "graphai";
+
+interface BrowserlessInputs {
+  url: string;
+  text_content?: boolean;
+}
+
+interface BrowserlessParams {
+  apiKey?: string;
+  debug?: boolean;
+  throwError?: boolean;
+}
+
+type BrowserlessResult =
+  | string
+  | {
+      onError?: {
+        message: string;
+        status?: number;
+        error: any;
+      };
+    };
+
+export const browserlessAgent: AgentFunction<BrowserlessParams, BrowserlessResult, BrowserlessInputs, DefaultConfigData> = async ({ namedInputs, params }) => {
+  const { url, text_content } = namedInputs;
+  const throwError = params?.throwError ?? false;
+
+  const browserlessEndpoint = "https://chrome.browserless.io";
+  const browserlessToken =
```

Comment:
It should be able to retrieve apiKey not only from params?.apiKey but also from config.apiKey.

https://github.com/receptron/graphai/blob/main/llm_agents/openai_agent/src/openai_agent.ts#L91-L94